apps/fluent-tester/src/FluentTester.tsx

```diff
@@ -73,7 +74,7 @@ const Header: React.FunctionComponent<HeaderProps> = React.memo((props) => {
         style={fluentTesterStyles.testHeader}
         variant="heroLargeSemibold"
         color={theme.host.palette?.TextEmphasis}
-        testID={BASE_TESTPAGE}
+        {...testProps(BASE_TESTPAGE)}
```

Comment:
I'd add a comment right here explaining that `testProps` has to be the last prop passed in. 

`{...testProps(BASE_TESTPAGE)}` // For Android E2E testing purposes, testProps must be passed in after accessibilityLabel.`